docs/sample-solutions/event-driven-apps/commerce-chatbot.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Conversational Azure chatbot for hotel reservations
+description: Proven solution for building a conversational chatbot for commerce applications with Azure Bot Service, Cognitive Services and LUIS, Azure SQL Database, and Application Insights.
+author: iainfoulds
+ms.date: 06/22/2018
+---
+# Conversational Azure chatbot for hotel reservations
+
+This sample solution is applicable to businesses that have a need for integrating a conversational chatbot into applications. In this solution, a C# chatbot is used for a hotel chain that allows customers to check availability and book accommodation through a web or mobile application.
+
+Example application scenarios include providing a way for customers to view hotel availability and book rooms, review a restaurant take-out menu and place a food order, or search for and order prints of photographs. Traditionally, businesses would need to hire and train customer service agents to respond to these customer requests, and customers would have to wait until a representative is available to provide assistance.
+
+By leveraging Azure services such as the Bot Service and Language Understanding or Speech API services, companies can assist customers and process orders or reservations with automated, scalable bots.
+
+## Potential use cases
+
+You should consider this solution for the following use cases:
+
+* View restaurant take-out menu and order food
+* Check hotel availability and reserve a room
+* Search available photos and order prints
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
+
+![Architecture overview of the Azure components involved in a conversational chatbot][architecture]
```

Comment:
This diagram doesn't use the standard Azure colors, should use the standard icon set.

Reply:
All artwork for these solutions is due to go through an artwork vendor for consistency with icon set. @david-stanford has more info.